index.html

```diff
@@ -67,5 +67,130 @@ <h5 class="small-text">I can help you build a product , feature or website Look
             </li>             
      </ul>               
     </section>
+    <section class="works">
+        <div class="title">
+           <h2>My Recent Works</h2>
```

Comment:
- [ ] Kindly note that you are required to stick as much to the design as possible. This includes typography and font sizes and colors. According to your stylesheets, the font size is set to 2.8rem which when converted roughly comes to above 44px. Kindly use the provided font size of 40px so as to match the provided Figma design.

**This is how this section currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/185756260-5aa65446-b85b-4838-aed9-595125e88e86.png)

**This is how it is supposed to look like:**

![image](https://user-images.githubusercontent.com/86318284/185756282-6bf3acee-33d5-42a5-8363-0509ed77435c.png)